### ISSUE TITLE
bug in nysiis modified/refined algorithm

### ISSUE THREAD

**Issue by vilchesalves** [Yomguithereal/talisman#180]
Hello, everyone.

I've found a bug with the modified/refined NYSIIS algorithm. Sorry I couldn't find any guidelines for bug report, but I'll try my best to give you something useful.

Following the definition here: http://www.dropby.com/NYSIIS.html

Specifically this:
![image](https://user-images.githubusercontent.com/22381818/105244597-a176a100-5bd5-11eb-8692-c9a1751ee128.png)

The use case my tester found was the following:
input ANANND
expected ANAD
actual ANAND

This can also be confirmed in [talisman full docs](https://yomguithereal.github.io/talisman/phonetics/#nysiis):
![image](https://user-images.githubusercontent.com/22381818/105245583-ee0eac00-5bd6-11eb-9d46-6074e1525588.png)

## Test case:
```js
it('converts repeated letter correctly', () => {
  const input = 'ANANND';
  const expected = 'ANAD';
  const nysiisConverted = toNysiis(input);

  expect(nysiisConverted).toBe(expected);
});
```

Cheers!

**Comment by Yomguithereal** [comment]
Hello @vilchesalves, thanks for noticing this bug, it seems I missed the "repeat this last step as necessary" bit. This should be fixed now on v1.1.4.

**Comment by vilchesalves** [comment]
> Hello @vilchesalves, thanks for noticing this bug, it seems I missed the "repeat this last step as necessary" bit. This should be fixed now on v1.1.4.

Beautiful! Such quick action! Thank you very much. <3
